Weather App/controller/ViewController.swift

```diff
@@ -4,17 +4,23 @@ import UIKit
 class ViewController: UIViewController {
```

Comment:
Could you name this ViewController? `WeathersListViewController` or something else

Reply:
👍